lab_3_keywords_textrank/main.py

```diff
@@ -2,8 +2,19 @@
 Lab 3
 Extract keywords based on TextRank algorithm
 """
+import csv
```

Comment:
revert

Reply:
i accidentally reverted these whole files to the empty state (no code at all), so that if i revert this right now, there will be massive errors in lab4 because of import
let's just pretend nothing happened and everything is ok with changes in lab3 🙏

Reply:
oh nevermind, somehow i managed to deal with it